packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1383,6 +1398,14 @@ export function _useFakeChatAdapters(args: _FakeChatAdapterArgs): _FakeChatAdapt
 // @beta
 export const useTeamsCallAdapter: (args: Partial<TeamsCallAdapterArgs>, afterCreate?: ((adapter: TeamsCallAdapter) => Promise<TeamsCallAdapter>) | undefined, beforeDispose?: ((adapter: TeamsCallAdapter) => Promise<void>) | undefined) => TeamsCallAdapter | undefined;
 
+// @public
+export interface VideoBackgroundImage {
+    // (undocumented)
+    key: string;
+    tooltipText?: string;
+    url: string;
```

Comment:
Could this be a blob object or it is just always string?

Reply:
It will always be a string, that is what replacement effect needs.